kyaml/go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776
 	k8s.io/kube-openapi v0.0.0-20210421082810-95288971da7e
+	sigs.k8s.io/yaml v1.2.0
```

Comment:
~Are you sure this dependency is ok? [This note](https://github.com/kubernetes-sigs/kustomize/blob/f61b075d3bd670b7bcd5d58ce13e88a6f25977f2/cmd/k8scopy/main.go#L6-L13) makes it sound like we kyaml can't depend on any k8s.io modules, but there's one on the line directly above. Maybe it means from k/k specifically, which is not the case for the one you're adding.~

NVM, the api and kustomize modules already depend on this.

Reply:
I'm not too sure what that note means because kyaml also depends on k8s.io/kube-openapi in the line above. Do you know what `cmd/k8scopy` is used for?

Reply:
It's used to vendor in some very stable k/k code that Kustomize can't depend on in the usual way. Specifically, it seems to be used in both the [config](https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/internal/commands/internal/k8sgen/k8scopy.yaml) and the [kyaml](https://github.com/kubernetes-sigs/kustomize/blob/master/kyaml/yaml/internal/k8sgen/k8scopy.yaml) modules.